### PR TITLE
[Testing] Profiteering 1.1.0.0

### PR DESCRIPTION
**Pull request by DecSgr01**
add Profiteering

### CODE REVIEW

**Comment by AinaSnow** [comment]
@aonyxbot rebuild

File: testing/live/Profiteering/manifest.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[plugin]
+repository = "https://github.com/DecSgr01/Profiteering.git"
+commit = "4717ea2081dd1db3778a15b129d5f46102b8297f"
+owners = [
+    "DecSgr01",
+]
+project_path = "src"
+changelog = """\
+- 修复菜单弹出
```
Comment (Loskh) [inline testing/live/Profiteering/manifest.toml:9]: Invalid End-Of-File found for multi-line string

**Comment by DecSgr01** [comment]
@aonyxbot rebuild

**Comment by DecSgr01** [comment]
@aonyxbot rebuild

**Pull request closed without merging**